vpr/src/base/vpr_api.cpp

```diff
@@ -688,7 +688,9 @@ RouteStatus vpr_route_flow(t_vpr_setup& vpr_setup, const t_arch& arch) {
         std::string graphics_msg;
         if (route_status.success()) {
             //Sanity check the routing
-            check_route(router_opts.route_type);
+            if (!router_opts.disable_check_route) {
+                check_route(router_opts.route_type, router_opts.quick_check_route);
+            }
```

Comment:
Looks like this also adds an option to disable check_route? Not sure where `router_opts.disable_check_route` is coming from (don't see it in this PR).